sorbet/rbi/shims/test_helper.rbi

```diff
@@ -0,0 +1,5 @@
+# typed: true
+
+module RubyLsp::TestHelper
+  include Kernel # for `raise`
```

Comment:
You can do `requires_ancestor { Kernel }` instead in the ruby file.

Reply:
Yes it's defined in https://github.com/sorbet/sorbet/blob/363435333ad9080b614e0c6473aa800b717c2d28/gems/sorbet-runtime/lib/types/helpers.rb#L64. Since ruby-lsp depends on `sorbet-runtime`, it'll work in add-ons. The feature flag is for the static checker only.